src/containers/commitments/index.tsx

```diff
@@ -98,11 +98,11 @@ function Commitments() {
           <Events />
         </div>
 
-        {/* <div className="red section">
+        <div className="red section">
           <IllustratedText imageSrc={trombiImageSrc} alignment="center" title="L'équipe" titleAlignment="center">
             <p>Voici l'équipe qui se cache derrière Cassiopée qui vous propose l'évenement du Gala UTT</p>
           </IllustratedText>
```

Comment:
Tu peux enlever le IllustratedText nan ? Il ne sert pas à grand chose 🤷
```suggestion
          <p>Voici l'équipe qui se cache derrière Cassiopée qui vous propose l'évenement du Gala UTT</p>
```